xcube/util/cmaps.py

```diff
@@ -477,11 +495,11 @@ def parse_cm_code(cm_code: str) -> tuple[str, Optional[Colormap]]:
         cm_base_name, _, _ = parse_cm_name(cm_name)
         n = len(cm_items)
         if cm_type == "categorical":
-            values: list[int] = []
+            values: list[float] = []
```

Comment:
That's wrong, please revert. Categorical values are integer indexes.

Reply:
float is needed, otherwise we get this error, when accessing the data:

Reply:
This has been resolved in the discussion below